README_SECURITY.md

```diff
@@ -54,17 +73,26 @@ The same by passing an environment variable when launching Docker:
 docker run ... -e extender.authentication.users=file:users/myusers.txt extender/extender;
 ```
 
-Example user file:
+#### Using a URL
+Example using `application.yml` where users are loaded from the content server from `https://www.mysite.com/extender-users`:
 
 ```
-bob = super5ecret,ROLE_WINDOWS,ROLE_LINUX,ROLE_MACOS,enabled
-may = top5ecret,ROLE_MACOS,enabled
+extender:
+    authentication:
+        users: https://www.mysite.com/extender-users
+```
+
+The same by passing an environment variable when launching Docker:
+
+```
+docker run ... -e extender.authentication.users=https://www.mysite.com/extender-users extender/extender;
```

Comment:
I don't see a change for run-local.sh?
How does this work with the current workflow? (I.e. when I build for Switch)

Reply:
I'll move the extender.authentication.platforms config to the task definition.